package.json

```diff
@@ -282,9 +282,9 @@
 		"*.{js,json,ts,tsx,yml,yaml}": [
 			"wp-scripts format"
 		],
-		"*.{js,ts,tsx}": [
+		"**/*.{js,ts,tsx,jsx}": [
 			"npm run prelint",
-			"wp-scripts lint-js"
+			"eslint --config .eslintrc.strict.js"
```

Comment:
`lint-js` does some things like [setting `--ext` to non-standard extensions like `.jsx`, etc](https://github.com/WordPress/gutenberg/blob/trunk/packages/scripts/scripts/lint-js.js). Are we losing that with these changes (i.e. risk that we're not actually linting a bunch of files)? Could we still use `wp-scripts lint-js` and pass the `--config` argument instead?